### ISSUE TITLE
Heroku not displaying webpage

### ISSUE THREAD

**Issue by ecussler** [EarlofSmith/sweet-solution#69]
Hey guys! Long time no see - hope you are doing well. I'm in the midst of the job search (yuck) and just going over some of the pages we worked on and notice that the Sweet Solutions page isn't working for me anymore. Are either of you having an issue with this? I'm using `https://sweet-solution.herokuapp.com ` as the url. 

Took a screen shot of the issue I'm having for reference: 

![Screen Shot 2023-10-03 at 2 52 35 PM](https://github.com/EarlofSmith/sweet-solution/assets/114117233/bf7fbb5f-ff86-4b42-9f52-00125a9e0cd2)

**Comment by EarlofSmith** [comment]
Yes it seems to not be working guessin its something to do with heroku?

**Comment by ecussler** [comment]
That’s my guess too - I know we used Todd’s heroku for that project and I’m
guessing he might’ve cancelled his account or something. I’ll look into it
today and see what I can do. Keep an eye out for a PR 👍🏼
